### PR TITLE
Slow stop services

### PR DESCRIPTION
**Pull request by obino**
This PR is to address the issues we have seen in slow start of autoscaled nodes.There are 2 parts in this PR (and I can split it if preferred):

- don't try to stop a non-running service: since we go to monit, at time the process can take much longer to actually do nothing;

- autoscaled node don't check for DB tables: this shave off about 1minute to the ready time for autoscaled nodes.

### CODE REVIEW

**Comment by obino** [comment]
OCD build http://ocd.appscale.net:8080/job/Daily%20Build/6652/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -4492,6 +4504,7 @@ def start_appcontroller(node)
   def start_admin_server
     Djinn.log_info('Starting AdminServer')
     script = `which appscale-admin`.chomp
+    HelperFunctions.log_and_crash("Cannod find appscale-admin!") if script.empty?
```
Comment (cdonati) [inline AppController/djinn.rb]: Cannot

File: AppController/lib/monit_interface.rb [hunk 2]
```diff
@@ -203,8 +209,9 @@ def self.service_config(process_name, group, start_cmd, env_vars, mem)
   end
 
   def self.is_running?(watch)
-    output = run_cmd("#{MONIT} summary | grep \"'#{watch}'\" | grep -E "\
-                     '"(Running|Initializing|OK)"')
+    script = `which appscale-admin`.chomp
+    HelperFunctions.log_and_crash("Cannod find appscale-admin!") if script.empty?
```
Comment (cdonati) [inline AppController/lib/monit_interface.rb]: Cannot

**Comment by obino** [comment]
This PR wis superseded by the move to systemd.

**Pull request closed without merging**